addons/debugger/addon.json

```diff
@@ -73,24 +73,20 @@
       "id": "log_invalid_cloud_data",
       "type": "boolean",
       "default": false
+    },
+    {
+      "name": "Faster graph rendering",
```

Comment:
This might sound like it's a setting to refresh the graph more frequently instead of removing the fill/animation.